wallet/res/values/strings-extra.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <resources>
 
+	<string name="onboarding_create_wallet">Create a New Wallet</string>
+	<string name="onboarding_recover_wallet">Restore Wallet From Passphrase</string>
```

Comment:
Change this to "Recovery phrase."

Recovery phrase is used in other places in the app.

Passphrase is technically something else, which this app doesn't support.